VotingApplication/VotingApplication.Web/Scripts/Controllers/Create/ConfirmStepController.js

```diff
@@ -67,5 +83,73 @@
                 return choice.Name;
             });
         }
+
+        function getValidInvitees() {
+            return $scope.newPoll.Invitations.filter(function (invite) {
+                // Maybe this needs to be pulled out of this controller
+                var emailRegEx = /^([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\.([a-z]{2,6}(?:\.[a-z]{2})?)$/i;
+                return emailRegEx.test(invite);
+            });
+        }
+
+        function formatUtcPollExpiry() {
+
+            var utcDate = getUtcPollExpiry();
+
+            if (utcDate) {
+                return moment(utcDate).format('YYYY/MM/DD - HH:mm');
+            } else {
+                return 'Invalid Date';
+            }
+        }
+
+        function getUtcPollExpiry() {
+            if (!$scope.newPoll.DoesExpire) {
+                return null;
+            }
+
+            var date;
+
+            if (!$scope.newPoll.ExpiryDate) {
```

Comment:
`$scope.newPoll.ExpiryDate` isn't clear if the date is being stored as UTC or local.